### ISSUE TITLE
Require user permission to use motionsensor.GetColourMaterial()

### ISSUE THREAD

**Issue by jorjic** [Facepunch/garrysmod-requests#1949]
### Details
The kinect can be used as a webcam with `motionsensor.GetColourMaterial`. The wiki description says:
> Returns the depth map material.

Which is wrong, since it actually returns an IMaterial with the user's kinect as a camera.

The purpose for this request is to require user consent via the new permissions library before allowing arbitrary access to use the kinect as a camera. This would not require consent for using the kinect as a motion capturing device (like how sandbox already supports with ragdolls).

**Comment by Kefta** [comment]
Can you still use the Kinect without crashing? Not saying to invalidate this request - this should still be done, but I didn't know it was functional.

**Comment by jorjic** [comment]
> Can you still use the Kinect without crashing?

Yeah, on the standard branch. I'm not sure how well it works on the beta branches.
